src/designer/elsa-workflows-studio/src/components/screens/workflow-instance-viewer/elsa-workflow-instance-viewer-screen/elsa-workflow-instance-viewer-screen.tsx

```diff
@@ -137,7 +137,8 @@ export class ElsaWorkflowInstanceViewerScreen {
 
   componentDidLoad() {
     if (!this.designer) {
-      this.designer = this.el.querySelector("elsa-designer-tree") as HTMLElsaDesignerTreeElement;
+      // this.designer = this.el.querySelector("elsa-designer-tree") as HTMLElsaDesignerTreeElement;
+      this.designer = this.el.querySelector('elsa-designer') as HTMLElsaDesignerTreeElement;
```

Comment:
Same as with the blueprint viewer screen, this needs to be somehow made configurable, otherwise everyone will suddenly use the new designer, whether they want to or not.